WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -85,6 +85,10 @@ class OrdersViewController: UIViewController {
         }
     }
 
+    /// Suppress error warnings when refreshing data in the background
+    ///
+    private var suppressErrorWarning: Bool = false
```

Comment:
This seems brittle. Is there a better way to handle the OrdersListViewController's data and state, when it's in the background?

Reply:
@mindgraffiti assuming we clear all of the order data like I suggested above ☝️ , we potentially don't need to do any of the updates here in `OrdersViewController`